admin/test-brew-recipe

```diff
@@ -1,11 +1,9 @@
 #!/usr/bin/env python
-# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+# Copyright 2015 ClusterHQ.  See LICENSE file for details.
 """
 Test a brew recipe generated by Buildbot.
 """
 
-from _preamble import TOPLEVEL, BASEPATH
```

Comment:
These are used in other scripts to provide context for error messages. Perhaps they should be kept and that context should be added to the relevant scripts./